grammars.yml

```diff
@@ -572,6 +572,8 @@ vendor/grammars/pig-latin:
 - source.pig_latin
 vendor/grammars/pike-textmate:
 - source.pike
+vendor/grammars/plugin-vscode:
+- source.bal
```

Comment:
@Shan1024 Just some helpful advice from a grammar author to another - the scopeName which uniquely identifies your grammar should be as unambiguous as possible,`.bal` works great as a file extension, but less so as a grammar ID. Remember, a name like this might easily end up conflicting with, say, `Behaviour Audit Language`... some machine-learning  language that gains traction in the future. Logically they're gonna name their grammar's ID  `bal`., too.

Reply:
@Alhadis: Thank you very much for the advice, I will rename the scope name.